ios/RNGestureHandler.m

```diff
@@ -58,6 +58,7 @@ CGRect RNGHHitSlopInsetRect(CGRect rect, RNGHHitSlop hitSlop) {
     return rect;
 }
 
+static NSHashTable<RNGestureHandler *> *allGestureHandlers;
```

Comment:
just wondering. Can't we do `[NSHashTable weakObjectsHashTable]` here?

Reply:
It won't compile because of "Initializer element is not a compile-time constant" error.